cluster/addons/fluentd-elasticsearch/es-image/Makefile

```diff
@@ -16,7 +16,7 @@
 
 # The current value of the tag to be used for building and
 # pushing an image to gcr.io
-TAG = v5.4.0-1
+TAG = v5.5.1
```

Comment:
Fine for me, just discovered it as I'm eagerly waiting for this to land and I started looking into the Makefile. 😁